docs/mxdoc.py

```diff
@@ -270,6 +270,8 @@ def _get_jupyter_notebook(lang, lines):
     for in_code, blk_lang, lines in _get_blocks(lines):
```

Comment:
`lines` from the function arguments is overwritten by the loop parameter.

Reply:
yes that's is actually the point the loop, To filter out the lines with <!--no-notebook-->